components/dropzone/overview.md

```diff
@@ -0,0 +1,75 @@
+---
+title: Overview
+page_title: DropZone Overview
+description: Discover the Blazor DropZone and explore the examples.
+slug: dropzone-overview
+tags: telerik,blazor,dropzone,external,drag,drop,file,overview
+published: True
+position: 0
+---
+
+# Blazor DropZone Overview
+
+The <a href="https://www.telerik.com/blazor-ui/dropzone" target="_blank">Blazor DropZone component</a> allows you to declare an external drop zone for an existing [FileSelect]({%slug fileselect-overview%}) or [Upload]({%slug upload-overview%}) component. This enables the users to drag and drop one or multiple files to a designated space in their viewport. Dropping the files in the connected DropZone area will automatically select/upload them.
+
+## Creating DropZone
+
+1. Use the `TelerikDropZone` tag to add the component to your razor page.
+1. Set the `Id` parameter of the DropZone component.
+1. Set the `DropZoneId` of the [FileSelect]({%slug fileselect-overview%}#fileselect-parameters)/[Upload]({%slug upload-overview%}#upload-parameters) component to match the `Id` of the DropZone. Read more in [DropZone Integration]({%slug dropzone-integration%}).
+1. (optional) Use the `NoteText` to add a note in the DropZone.
+
+>caption Basic configuration of the DropZone with FileSelect.
+````CSHTML
+<TelerikDropZone Id="@DropZoneId" NoteText="@NoteText" />
+
+<TelerikFileSelect DropZoneId="@DropZoneId"/>
+
+@code {
+    private string DropZoneId => "my-dropzone";
+
+    private string NoteText => "Allowed file types: DOCX and PDF";
```

Comment:
This note doesn't match the FileSelect configuration. Better remove it or adjust the example.